### ISSUE TITLE
Assertion violation at contractor/contractor_worklist_fixpoint.cc:99 (--worklist-fixpoint)

### ISSUE THREAD

**Issue by rainoftime** [dreal/dreal4#198]
Hi, for the following formula,
~~~~
(set-logic QF_NRA)
(declare-fun _substvar_12_ () Real)
(declare-const r5 Real)
(check-sat)
(assert (= _substvar_12_ r5))
(check-sat)
(declare-const r10 Real)
(declare-const v15 Bool)
(assert (and (<= 0.7 r10) v15))
(check-sat)
~~~~

dreal 0d9a0a5 throws an assertion violation
~~~~
delta-sat with delta = 0.001
delta-sat with delta = 0.001
dreal: dreal/contractor/contractor_worklist_fixpoint.cc:99: virtual void dreal::ContractorWorklistFixpoint::Prune(dreal::ContractorStatus*) const: Assertion `static_cast<size_t>(branching_point) < input_to_contractors_.size()' failed.
Aborted (core dumped)
~~~~

**Comment by rainoftime** [comment]
There are no push/pop operations

**Comment by soonho-tri** [comment]
Thanks for the report. Fixed by a9a3d7f6b7acba6ebab8e416e6f041889ae99472.
